variables.tf

```diff
@@ -181,3 +193,22 @@ variable "wiki_enabled" {
   default     = false
   description = "Enable wiki for the project"
 }
+
+variable "pipeline_schedule" {
```

Comment:
```
variable "pipeline_schedule" {
  type = object({
    active         = optional(bool, true)
    cron           = string
    cron_timezone  = optional(string, "UTC")
    description    = string
    ref            = optional(string, "refs/heads/main")
    take_ownership = optional(bool, false)
  })
  default = null
}
```

Modified 2 things:
* https://registry.terraform.io/providers/gitlabhq/gitlab/latest/docs/resources/pipeline_schedule
cron, description, project, ref are required variables.
* according to the docs `main` as `ref` is not correct this should be `refs/heads/main`.
* Refactored default into the optionals, otherwise when you set 1 variable the default gets fully overwritten. The approach above prevents that.